### ISSUE TITLE
about how to generate density maps

### ISSUE THREAD

**Issue by leizhu1989** [shizenglin/Deep-NCL#6]
hello! 
I want to konw how to make the last layer of the output into density maps ?
could you give me some ideas? thank you very much

**Comment by shizenglin** [comment]
I'm not really understand your question. Basically, to train your model, you need first generate your density map ground truth. Then you need design your network, in which the output of last layer should have the same size with your density map ground truth. If your question is about how to get the final estimated density map of our deep NCL, I can say that the final estimated density map is obtained by averaging all outputs of deep NCL. I hope I answer you well.

**Comment by leizhu1989** [comment]
thank you ,I should make clear my question. 
I want to know whether the featuremap of the last layer of convolutional output can be directly drawn into an density maps image ,when I want to write a demo to test a picture in caffe project.

**Comment by shizenglin** [comment]
Yes, as you said, the last layer of convolutional output can be directly drawn into an density maps image. It would be better to understand if you take some time to read the codes.

**Comment by leizhu1989** [comment]
thank you ，I got it
